setup.py

```diff
@@ -49,12 +49,12 @@
 
 EXTRAS_REQUIRE = {
     'doc': [
-        'sphinx==1.8.5',
+        'sphinx==3.2',
         'nbsphinx==0.8.5',
         'nbsphinx-link==1.3.0',
         'pandas==0.23.0',
         'pvlib==0.7.1',
-        'sphinx_rtd_theme==0.4.3',
+        'sphinx_rtd_theme==0.5.0',
```

Comment:
Bumping `sphinx_rtd_theme` was necessary to have nice rendering with the newer sphinx version